tests/unit/test_distro.py

```diff
@@ -252,18 +252,8 @@ def test__get_token_from_service_key_ok(self, mocker):
         )
         assert distro.SolarWindsDistro()._get_token_from_service_key() == "foo-token"
 
-    def test__configure_logs_export_env_defaults_invalid_protocol(self, mocker):
```

Comment:
This test and its metrics/traces equivalents removed because protocol checks are now done by `_configure`.